src/files/type-from-ext/extensions/audio.json

```diff
@@ -12,7 +12,6 @@
   "aac",
   "mp2",
   "mp3",
```

Comment:
As we want to look up a file type from an encoding, a map or object would be a better mechanism for storing these lookups.

extToType.js

```js
{
  amr: 'audio',
  bw: 'audio',
  jpg: 'image',
  jpeg: 'image',
  mp4: 'video'
  // ...and on and on
}
```

This ensures that we don't put the same extension in more than one file and keeps lookups as fast as possible and allows us to easily add new icon types for specific extensions if we need to.